lib/modules/powershell/collection/PiiParser.py

```diff
@@ -0,0 +1,93 @@
+from lib.common import helpers
+import base64
+
+class Module:
+
+    def __init__(self, mainMenu, params=[]):
+
+        # metadata info about the module, not modified during runtime
+        self.info = {
+            # name for the module that will appear in module menus
+            'Name': 'PiiParser',
+
+            # list of one or more authors for the module
+            'Author': ['@utkusen, gozdesinturk'],
+
+            # more verbose multi-line description of the module
+            'Description': ('Parses PII data from given Word/Excel document and returns the first few characters of data.'),
+
+            # True if the module needs to run in the background
+            'Background' : True,
+
+            'SaveOutput' : False,
+
+            # File extension to save the file as
+            'OutputExtension' : None,
+
+            # True if the module needs admin rights to run
+            'NeedsAdmin' : False,
+
+            # True if the method doesn't touch disk/is reasonably opsec safe
+            'OpsecSafe' : True,
+
+            'Language' : 'powershell',
+
+            'MinLanguageVersion' : '2',
+
+            # list of any references/other comments
+            'Comments': [
+                ''
+            ]
+        }
+
+        # any options needed by the module, settable during runtime
+        self.options = {
+            # format:
+            #   value_name : {description, required, default_value}
+            'Agent' : {
+                # The 'Agent' option is the only one that MUST be in a module
+                'Description'   :   'Agent to run the module on.',
+                'Required'      :   True,
+                'Value'         :   ''
+            },
+            'FilePath' : {
+                'Description'   :   'Path of file to search.',
+                'Required'      :   True,
+                'Value'         :   ''
+            }
+        }
+
+        # save off a copy of the mainMenu object to access external functionality
+        #   like listeners/agent handlers/etc.
+        self.mainMenu = mainMenu
+
+
+        if params:
+            for param in params:
+                # parameter format is [Name, Value]
+                option, value = param
+                if option in self.options:
+                    self.options[option]['Value'] = value
+
+
+    def generate(self, obfuscate=False, obfuscationCommand=""):
+
+        moduleSource = self.mainMenu.installPath + "/data/module_source/collection/Get-PII.ps1"
+        try:
+            f = open(moduleSource, 'r')
+        except:
+            print helpers.color("[!] Could not read module source path at: " + str(moduleSource))
```

Comment:
Update to `print (helpers.color("[!] Could not read module source path at: " + str(moduleSource)))`